common/selectors/wallet.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import type { State } from 'reducers';
+import { BaseWallet } from 'libs/wallet';
+import { getNetworkConfig } from 'selectors/config';
+import Big from 'big.js';
+import type { Token } from 'config/data';
+
+export function getWalletInst(state: State): ?BaseWallet {
+  return state.wallet.inst;
```

Comment:
It is not wallet, it is state.wallet. Install flow, click on ctrl-click on State :)